addons/scene_object_brush/plugin.cfg

```diff
@@ -3,5 +3,5 @@
 name="Scene Object Brush"
 description="Scene Object brush is a 3D addon drawing tool for the Godot game engine. It's meant to ease the placement of scenes (ref Unity prefab drawers)"
 author="sacristan"
-version="1.0.1"
```

Comment:
please revert changes to this since this is not in main yet. Also this is not a patch :) Ill increment this once there's a new release